cmd/systray/systray.go

```diff
@@ -74,6 +78,12 @@ func init() {
 
 	desktop := strings.ToLower(os.Getenv("XDG_CURRENT_DESKTOP"))
 	switch desktop {
+	case "gnome":
+		// GNOME expands submenus downward in the main menu, rather than flyouts to the side.
+		// Either as a result of that or another limitation, there seems to be a maximum depth of submenus.
+		// Mullvad countries that have a city submenu are not being rendered, and so can't be selected.
+		// Handle this by simply treating all mullvad countries as single-city and select the best peer.
+		hideMullvadCities = true
```

Comment:
this changes gnome's menu delay from 10ms to 0 ... is that intentional?

Reply:
oh, good catch.  In this case that should be fine, as GNOME doesn't use libdbusmenu which is where the problem seems to lie.